src/lib/egoJwt.ts

```diff
@@ -21,9 +21,8 @@
 import memoize from 'lodash/memoize';
 
 import createEgoUtils from '@icgc-argo/ego-token-utils';
-import { getAppConfig } from '@/global/config';
 
-const TokenUtils = createEgoUtils(getAppConfig().EGO_PUBLIC_KEY);
+const TokenUtils = createEgoUtils('');
 
 export const decodeToken = memoize((egoJwt?: string) =>
 	egoJwt ? TokenUtils.decodeToken(egoJwt) : null,
```

Comment:
this a little more involved to get working with the new app config.
none of these functions appear to be in use, so may be a non issue
@demariadaniel are these needed?

Reply:
changed to empty string to not block, did not remove until verifying not needed

Reply:
I see what you mean, this was setting up for additional Login functionality but aren't in use at the moment